pkg/store/bucket.go

```diff
@@ -1627,3 +1627,25 @@ func (s queryStats) merge(o *queryStats) *queryStats {
 
 	return &s
 }
+
+// GetMetas gets all bucket metas
+func GetMetas(ctx context.Context, logger log.Logger, bkt objstore.Bucket) ([]*block.Meta, error) {
```

Comment:
Why do you think this name is better? I feel personally that `DownloadMetas` makes more sense, as you show that it is complex thing, not getting something from memory. 

Also getters for Golang never have `Get` prefix and it looks like a getter with this prefix (:

Reply:
Download means we put this data on our local disk, so here that's confusing, imo, Get is more suitable, the same as the method `Get` in the `BucketReader` interface

Reply:
hm, can agree